caput/astro/units.py

```diff
@@ -0,0 +1,50 @@
+"""A set of useful constants and conversions in Astronomy and Cosmology.
```

Comment:
It would be better to call this `caput.astro.constants`.  Its scope has broadened from just being unit definitions.